service/src/main/java/tds/session/web/resources/PauseSessionResponseResource.java

```diff
@@ -14,24 +16,41 @@
 /**
  * A HATEOAS representation of the {@link PauseSessionResponse}.
  */
-// TODO:  This is returning too much data: should return status, datechanged, dateended, collection of affected examids, and link to session.
 public class PauseSessionResponseResource extends ResourceSupport {
-    private final PauseSessionResponse pauseSessionResponse;
+
+    private String status;
+
+    private Instant dateChanged;
+
+    private Instant dateEnded;
+
+    List<UUID> examIds;
 
     public PauseSessionResponseResource(PauseSessionResponse pauseSessionResponse) {
-        this.pauseSessionResponse = pauseSessionResponse;
-        Resources<UUID> examIdResources = new Resources<>(pauseSessionResponse.getExamIds());
-        examIdResources.add(linkTo(
-                methodOn(SessionController.class)
-                        .findSessionById(pauseSessionResponse.getSession().getId())
-        ).withRel("session"));
+        Session session = pauseSessionResponse.getSession();
+        this.status = session.getStatus();
+        this.dateChanged = session.getDateChanged();
+        this.dateEnded = session.getDateEnd();
+        this.examIds = pauseSessionResponse.getExamIds();
         this.add(linkTo(
                 methodOn(SessionController.class)
-                .findSessionById(pauseSessionResponse.getSession().getId())
+                .findSessionById(session.getId())
                 ).withRel("session"));
     }
 
-    public PauseSessionResponse getPauseSessionResponse() {
-        return pauseSessionResponse;
+    public String getStatus() {
```

Comment:
Ugh - you're right.  The `PauseSessionResponse` in the `client` should probably be updated to match what the resource looks like.